egs2/mediaspeech/asr1/README.md

```diff
@@ -0,0 +1,3 @@
+### Mediaspeech Corpus
+
+To run for a language other than Spanish, please set the `lang` flag in `local/data.sh` with the language tag (ES, TR, AR, FR).
```

Comment:
Try to use a three-letter language code instead of a two-letter code.

Reply:
Actually, the zipped datasets are named after these two-letter codes (TR.zip, etc.) Therefore, I think including both could be less confusing. Or I can write: ES for Spanish, TR for Turkish, etc? Please let me know if you agree to this style.

Reply:
oh, I see.
Your way is better then.
Sorry